.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor
 composer.lock
+.idea/
```

Comment:
This should be part of the `.gitignore` of your PC.

```
echo '.idea/' >> ~/.gitignore
```

This kind of configuration is too related to your IDE.